app/main.py

```diff
@@ -1 +1,28 @@
-# write your code here
+from __future__ import annotations
+from app.players.dwarves.dwarf_warrior import DwarfWarrior
+from app.players.dwarves.dwarf_blacksmith import DwarfBlacksmith
+from app.players.elves.druid import Druid
+from app.players.elves.elf_ranger import ElfRanger
+
+
+def calculate_team_total_rating(
+        team: list
+        [Druid | DwarfBlacksmith | DwarfWarrior | ElfRanger]
+) -> int:
+    total_rating = 0
+    for player in team:
+        total_rating += player.get_rating()
+    return total_rating
+
+
+def elves_concert(elves: list[ElfRanger | Druid]) -> None:
```

Comment:
You can use Elf  like inner type of list
```suggestion
def elves_concert(elves: list[Elf]) -> None:
```